models/lorenz_04/README.rst

```diff
@@ -1,899 +1,143 @@
-    <TD>Location to which to interpolate.</TD></TR>
-
-<TR><TD valign=top><em class=code>itype</em></TD>
-    <TD>Integer indexing which type of state variable is to be 
-        interpolated. Is ignored for this model since there
-        is only a single type of variable.</TD></TR>
-
-<TR><TD valign=top><em class=code>obs_val</em></TD>
-    <TD> The interpolated value from the model.</TD></TR>
-
-<TR><TD valign=top><em class=code>istatus</em></TD>
-    <TD>Quality control information about the observation of 
-        the model state. 0 == successful.</TD></TR>
-
-</TABLE>
-
-</div>
-<br />
-
-<!--===================== DESCRIPTION OF A ROUTINE =====================-->
-
-<A NAME="get_model_time_step"></A>
-<br />
-<div class=routine>
-<em class=call>var = get_model_time_step()</em>
-<pre>
-type(time_type) :: <em class=code>get_model_time_step</em>
-</pre>
-</div>
-
-<div class=indent1>
-<!-- Description -->
-
-<P>
-Returns the time step (forecast length) of the model;
-The time step defaults to 1 hour but is settable by namelist.
-</P>
-
-<TABLE width=100% border=0 summary="" cellpadding=3>
-
-<TR><TD valign=top><em class=code>var&nbsp;&nbsp;&nbsp;</em></TD>
-    <TD>Smallest time step of model.</TD></TR>
-
-</TABLE>
-
-</div>
-<br />
-
-<!--===================== DESCRIPTION OF A ROUTINE =====================-->
-
-<A NAME="static_init_model"></A>
-<br />
-<div class=routine>
-<em class=call>call static_init_model()</em>
-</div>
-
-<div class=indent1>
-<!-- Description -->
-
-<P>
-Reads the namelist, allocates space for the state vector,
-initializes the locations, generates some initial parameters.
-</P>
-
-<TABLE width=100% border=0 summary="" cellpadding=3>
-</TABLE>
-
-</div>
-<br />
-
-<!--===================== DESCRIPTION OF A ROUTINE =====================-->
-
-<A NAME="end_model"></A>
-<br />
-<div class=routine>
-<em class=call>call end_model()</em>
-</div>
-
-<div class=indent1>
-<!-- Description -->
-
-<P>
-A stub routine in this model.
-</P>
-
-<TABLE width=100% border=0 summary="" cellpadding=3>
-</TABLE>
-
-</div>
-<br />
-
-<!--===================== DESCRIPTION OF A ROUTINE =====================-->
-
-<A NAME="init_time"></A>
-<br />
-<div class=routine>
-<em class=call>call init_time(time)</em>
-<pre>
-type(time_type), intent(out) :: <em class=code>time</em>
-</pre>
-</div>
+Model Formulation
+-----------------
 
-<div class=indent1>
-<!-- Description -->
+For Lorenz 05, DART to advances the model, gets the model state and metadata
+describing this state, finds state variables that are close to a given
+location, and does spatial interpolation for model state variables.
 
-<P>
-Returns the time at which the model will start 
-if no input initial conditions are 
-to be used. This is used to spin-up the model from rest.
-Sets time to 0 in this model.
-</P>
+Equations
+~~~~~~~~~
 
-<TABLE width=100% border=0 summary="" cellpadding=3>
+The equations are in the README (the non-reStructuredText version of it).
```

Comment:
I am thinking of converting the README into  bob.rst and then blending the two.  The equations I may just keep as ASCII ART in a code block. The monospacing should preserve the 'art'.